### ISSUE TITLE
Remove references to apitest.dataverse.org in guides (demo server instead)

### ISSUE THREAD

**Issue by pdurbin** [IQSS/dataverse#3345]
https://apitest.dataverse.org was set up during the Dataverse 4.0 rewrite as a place for API developers to confirm that we had maintained backward compatibility, especially for the SWORD API (#780). At the time, we had concerns with the stability of the API but now we're the API is in production and we're quite comfortable with API users hitting both Harvard's production instance of Dataverse ( https://dataverse.harvard.edu ) and the demo site ( https://demo.dataverse.org ).

At this point, keeping https://apitest.dataverse.org up to date with the latest release is a bit of an administrative burden so I plan to update the line below from http://guides.dataverse.org/en/4.5/api (via a pull request) to indicate that https://demo.dataverse.org should now be used for testing rather than the apitest server:

"Rather than using a production installation of Dataverse, API users should use http://apitest.dataverse.org for testing."

I'll also communicate with the community about this change. Then we'll shut down the apitest server or at least uninstall Dataverse from it.

**Comment by pdurbin** [comment]
I just posted "apitest.dataverse.org to be shut down, use demo.dataverse.org instead" as a head up to the community at https://groups.google.com/d/msg/dataverse-community/5r1_dB5WShQ/mofiiE6VAwAJ

**Comment by pdurbin** [comment]
Fixed in in pull request #3355. I moved this issue to Code Review at https://waffle.io/IQSS/dataverse but it could probably go straight to QA. I'm comfortable merging it right now. It's not a code change. Just cleanup of the docs (especially removing the reference to the apitest server at http://guides.dataverse.org/en/4.5/api/ ) and cleanup of duplicate scripts that now are used by the phoenix server (automated deployment) rather than the apitest server.

**Comment by djbrooke** [comment]
Moving to QA

**Comment by pdurbin** [comment]
I opened https://github.com/IQSS/dataverse-client-python/issues/36 and left a comment at https://github.com/IQSS/dataverse-client-r/issues/2#issuecomment-247112442 to let other developers know about the change.

**Comment by djbrooke** [comment]
Looks good. In the documentation, I see that instances of apitest have been changed up to demo.

**Comment by pdurbin** [comment]
Pull request #3355 has been merged so I'm tagging this with the milestone for the next planned release (4.5.1). I assume this issue should be closed.

**Comment by pdurbin** [comment]
Note to self: I just deployed v. 4.5.1 build file-metadata-provenance-2295-f5de5cb to https://apitest.dataverse.org (dropping the database first) but I'd like to get away from using that hostname since it's still mentioned in pre-4.5.1 guides as a good place to test: "Rather than using a production installation of Dataverse, API users should use http://apitest.dataverse.org for testing." http://guides.dataverse.org/en/4.5/api

The server is certainly **NOT** a good place for API testing anymore because it contains potentially buggy pre-release code. My plan is to rename the server to "dev2" or something, some day.

Also, before rebuilding this server, I tried to preserve a file mentioned in #2803 but failed. I asked the user to upload it to the issue, if possible.
